test/battle/hold_effect/metronome.c

```diff
@@ -0,0 +1,40 @@
+#include "global.h"
+#include "test/battle.h"
+
+#define METRONOME_TURNS 6
```

Comment:
I would use an `ARRAY_COUNT` of `MetronomeMultipliers`

Reply:
it's a bit weird, because that array actually has one less entry than turns.

Reply:
Then let's expand the array to have to also have the x1.0 multiplier :)

Reply:
(Alternatively, you can do ARRAY_COUNT - 1)

Reply:
'aight, did that